examples/imperative-style-foreach-loops.flix

```diff
@@ -0,0 +1,184 @@
+///
+/// Flix supports writing code in an imperative
+/// style if it is preferable to you.
+///
+/// One of the features related to this is a
+/// foreach-loop.
+///
+/// In this example we explore how this loop
+/// works in Flix.
+///
+def main(): Unit & Impure =
```

Comment:
should be IO

Reply:
Yes, but this is the one I'm suggesting we remove (that's why I didn't touch it) :)